build/types/dash

```diff
@@ -6,3 +6,6 @@
 +../../lib/dash/segment_base.js
 +../../lib/dash/segment_list.js
 +../../lib/dash/segment_template.js
+
++../../lib/media/mp4_segment_index_parser.js
++../../lib/media/webm_segment_index_parser.js
```

Comment:
Maybe we should move those files to dash folder to make it clear?

Reply:
done!